server/src/utils.ts

```diff
@@ -38,6 +38,10 @@ const utils = {
     return false
   },
 
+  defineExpirationDate(oldDate: number) {
+    return new Date(oldDate + 60 * 60 * 24 * 30 * 1000).toISOString()
```

Comment:
there are two very seriously wrong things with this function

1. you should never use magic numbers like this. at the very least you should do something like `const MEANINGFUL_VARIABLE_NAME =  60 * 60 * 24 * 30 * 1000`. there is even a whole article about this on wikipedia: https://ru.wikipedia.org/wiki/%D0%9C%D0%B0%D0%B3%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%BE%D0%B5_%D1%87%D0%B8%D1%81%D0%BB%D0%BE_(%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5) and this is one of the hallmark signs of unclean programming. so please don't do this :)
2. if you have a function that returns a date type, try to make it return a date type, not an ISOString. if i called a function that was called `calculateDate`, i would expect it not to return a string. that would be very strange. for example, i give you a function called `calculateTotalCost`, and it gives you a string. does that make sense? no.